robottelo/ui/base.py

```diff
@@ -57,7 +57,7 @@ def find_element(self, locator):
             return None
 
     def search_entity(self, element_name, element_locator, search_key=None,
-                      katello=None, timeout=None):
+                      katello=None, timeout=20):
```

Comment:
Why is `timeout` being given a value of 20 here? A bit further down in the method, this call is made:

```python
            if timeout:
                element = self.wait_until_element(
                    (element_locator[0], element_locator[1] % element_name),
                    timeout=timeout,
                )
            else:
                element = self.wait_until_element(
                    (element_locator[0], element_locator[1] % element_name))
```

And the signature of `wait_until_element` looks like this:

    wait_until_element(self, locator, timeout=12, poll_frequency=0.5):

Changing `timeout=None` to `timeout=20` doesn't tweak the length of the default timeout. It changes the logical path taken by `search_entity`. The logical path taken by `search_entity` is something that a lot of code may depend on.